support-docs/README.md

```diff
@@ -7,7 +7,7 @@ If you are working on the site, or want to change text or other content, here is
 
 1. Start by reading the information in [site-organization](./site-organization), which summarizes how the size is organized.
 
-2. Next, consult the file [how-to-run-development-server](./how-tos/how-to-run-development-server.md) for an explanation of how to run a local development server so that you can preview the site on your personal computer and see the changes updated live as you make edits.
+2. Next, consult the file [how-to-run-development-server](./how-tos/how-the-development-site-works.md) for an explanation of how to run a local development server so that you can preview the site on your personal computer and see the changes updated live as you make edits.
```

Comment:
Should we update the title as well?

Reply:
I just realized, that I used the wrong link. The correct link is: how-to-run-local-copy.md, but does not contain much information. Yes the title should also be updated. The correct change is 
```
[how-to-run-local-copy](./how-tos/how-to-run-local-copy.md)
```
Could you apply this, I did this via the github page and cannot modify my patch here without moving everything to a fork :/

Reply:
Since this is the only change, I just made it its own PR.  Closing this one in favor of that one.